colaraz/cms/templates/widgets/header.html

```diff
@@ -302,12 +305,14 @@ <h2 class="sr-only">${_("Account Navigation")}</h2>
                     <span class="icon-bar bottom-bar"></span>
                 </button>
                 <ul>
-                    <li><a href="#"><span class="ecosystem-link"></span>Ecosystem</a></li>
-                    <li><a href="//${'{}.{}'.format(site_identifier, settings.COLARAZ_NAVIGATION_URLS.get('RECRUITMENT_URL'))}"><span class="recruiting-link"></span>Recruiting</a></li>
-                    <li><a href="//${'{}.{}'.format(site_identifier, settings.COLARAZ_NAVIGATION_URLS.get('MANAGEMENT_URL'))}"><span class="projects-link"></span>Management</a></li>
-                    <li><a href="#"><span class="talent-link"></span>Performance</a></li>
-                    <li class="active"><a href="//${'{}.{}'.format(site_identifier, settings.COLARAZ_NAVIGATION_URLS.get('DEVELOPMENT_URL'))}"><span class="development-link"></span>Development</a></li>
-                    <li><a href="//${'{}.{}'.format(site_identifier, settings.COLARAZ_NAVIGATION_URLS.get('SETTINGS_URL'))}"><span class="settings-link"></span>Settings</a></li>
+                  <li><a href="//${'{}.{}/cvsocial/activity?{}'.format(site_identifier, settings.COLARAZ_SUB_DOMAIN, role_based_urls.get('ecosystem'))}"><span class="ecosystem-link"></span>Ecosystem</a></li>
+                    <li><a href="//${'{}.{}/{}'.format(site_identifier, settings.COLARAZ_SUB_DOMAIN, role_based_urls.get('recruiting'))}"><span class="recruiting-link"></span>Recruiting</a></li>
```

Comment:
I think we can create some constants for these endpoint names in our colaraz code to avoid any spelling mistakes?

Like: 
```
RECRUITING= 'recruiting'
ECOSYSTEM= 'ecosystem'
```
or 
```
COLARAZ_APP_LINKS = {
    "RECRUITING": "recruiting",
    "ECOSYSTEM": "ecosystem",
    .....
}
```

Reply:
Its a dict and if we are entering the wrong key then we can also enter misspelled variable name. I don't think it makes much of a difference.